src/main/resources/i18n/messages.properties

```diff
@@ -462,6 +462,7 @@ settings.sounds.enable = Enable Sounds
 settings.fa.title = Forged Alliance Forever
 settings.fa.gameConfiguration = Game configuration
 settings.fa.autoDownloadMaps = Automatic Map Download
+settings.fa.forceAffinity = Force CPU Affinity
```

Comment:
Maybe in the UI it should be called 'Adjust CPU affinity', the use of 'Force' sounds so harsh.

Reply:
Adjust isn't very clear what a toggle does. I think Force is fine